src/main/java/net/sf/jabref/logic/l10n/Localization.java

```diff
@@ -15,7 +15,6 @@
 
     public static final String RESOURCE_PREFIX = "l10n/JabRef";
     public static final String MENU_RESOURCE_PREFIX = "l10n/Menu";
-    public static final String BIBTEX = "BibTeX";
```

Comment:
It was only used in the viewModel where I changed it to Localization.lang("BibTex").

On Thu, Sep 08, 2016 at 10:30 PM, Oliver Kopp < notifications@github.com [notifications@github.com] > wrote:
In src/main/java/net/sf/jabref/logic/l10n/Localization.java [https://github.com/JabRef/jabref/pull/1940#discussion_r78083018] :

> @@ -15,7 +15,6 @@
>  
>      public static final String RESOURCE_PREFIX = "l10n/JabRef";
>      public static final String MENU_RESOURCE_PREFIX = "l10n/Menu";
> -    public static final String BIBTEX = "BibTeX";


Why did you remove that?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub [https://github.com/JabRef/jabref/pull/1940/files/e5277fac194f6ec75df9d693b4c5845511a679a6#r78083018] , or mute the thread [https://github.com/notifications/unsubscribe-auth/AOoU1WoyuZFMXSbh5mdWxJ4KqeHLp2gUks5qoHBvgaJpZM4J4D-T] .